deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -377,96 +554,107 @@ services:
       - storage-system
       - server
     environment:
-      OCIS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
-      OCIS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
-      OCIS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+      STORAGE_SYSTEM_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      STORAGE_SYSTEM_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      STORAGE_SYSTEM_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
-      STORAGE_HOME_DATAPROVIDER_INSECURE: "${INSECURE:-false}"
+      STORAGE_SYSTEM_DATAPROVIDER_INSECURE: "${INSECURE:-false}"
 
       STORAGE_SYSTEM_GRPC_ADDR: 0.0.0.0:9215
       STORAGE_SYSTEM_HTTP_ADDR: 0.0.0.0:9216
       STORAGE_SYSTEM_DATA_SERVER_URL: http://storage-system:9216/data
 
-      STORAGE_JWT_SECRET: ${OCIS_METADATA_JWT_SECRET:-Pive-Fumkiu4}
-      STORAGE_TRANSFER_SECRET: ${STORAGE_TRANSFER_SECRET:-replace-me-with-a-transfer-secret}
+      STORAGE_SYSTEM_JWT_SECRET: ${OCIS_JWT_SECRET}
+      STORAGE_SYSTEM_TRANSFER_SECRET: ${STORAGE_TRANSFER_SECRET}
```

Comment:
according to https://owncloud.dev/extensions/storage-system/configuration/, `STORAGE_SYSTEM_TRANSFER_SECRET` does not exist

Reply:
same as above